tests/integration/test_data_store_mgr.py

```diff
@@ -131,6 +131,37 @@ async def harness(mod_flow, mod_scheduler, mod_start):
         yield schd, data
 
 
+@pytest.fixture(scope='module')
+async def harness_prune(mod_flow, mod_scheduler, mod_start):
```

Comment:
Please can we have  at least one line descriptive docstring?

Reply:
This "harness" has been removed, and included in the prune test scope (as per Ronnie's comment)..